src/components/contexts/root-context/RootContext.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { domains } from '@site/src/data-storage/domains';
+
+const MainContext = React.createContext();
+
+export const useRootContext = () => {
+    return React.useContext(MainContext);
+}
+
+export const RootContext = ({ children }) => {
+    const [is_deriv_location, setIsDerivLocation] = React.useState(true);
```

Comment:
For now let's remove the `is_deriv_location` from context, let's keep it simple. 
please remove the `domains.js` file as well. thanks thanks.